docusaurus.config.js

```diff
@@ -100,57 +100,61 @@ module.exports = {
-            {
-              label: 'SAP Cloud SDK for Java',
-              to: 'docs/java/getting-started'
+              label: 'Tutorials',
+              href: 'https://developers.sap.com/tutorial-navigator.html?tag=products:technology-platform/sap-cloud-sdk/sap-cloud-sdk'
             },
             {
-              label: 'SAP Cloud SDK for JS',
-              to: 'docs/js/getting-started'
+              label: 'Maven',
+              href: 'https://search.maven.org/search?q=g:com.sap.cloud.sdk*'
             },
             {
-              label: 'SAP Cloud SDK Tutorials',
-              href: 'https://developers.sap.com/tutorial-navigator.html?tag=products:technology-platform/sap-cloud-sdk/sap-cloud-sdk'
+              label: 'Support',
+              to: 'docs/java/support-java'
             }
           ]
         },
         {
-          title: 'Additional Resources',
+          title: 'SAP Cloud SDK for JavaScript',
           items: [
             {
-              label: 'Stack Overflow',
-              href: 'https://stackoverflow.com/questions/tagged/sap-cloud-sdk'
+              label: 'Tutorials',
+              to: 'docs/js/tutorials/getting-started/introduction'
             },
             {
-              label: 'SAP Developers Portal',
-              href: 'https://developers.sap.com/topics/cloud-sdk.html'
+              label: 'npm',
+              href: 'https://www.npmjs.com/search?q=%40sap-cloud-sdk'
             },
             {
-              label: 'SAP Cloud SDK Community Page',
-              href: 'https://community.sap.com/topics/cloud-sdk'
+              label: 'GitHub',
+              href: 'https://github.com/SAP/cloud-sdk-js'
             },
             {
-              label: 'SAP Cloud SDK on SAP Help',
-              href: 'https://help.sap.com/viewer/product/SAP_CLOUD_SDK/1.0/en-US?task=discover_task'
+              label: 'Support',
+              to: 'docs/js/support-js'
             }
           ]
         },
         {
-          title: 'Package Repositories',
+          title: 'Additional Resources',
           items: [
             {
-              label: 'SAP Cloud SDK for JavaScript',
-              href: 'https://www.npmjs.com/search?q=%40sap-cloud-sdk'
+              label: 'Stack Overflow',
+              href: 'https://stackoverflow.com/questions/tagged/sap-cloud-sdk'
             },
             {
-              label: 'SAP Cloud SDK for Java',
-              href: 'https://search.maven.org/search?q=g:com.sap.cloud.sdk*'
+              label: 'SAP Cloud SDK Community Page',
+              href: 'https://community.sap.com/topics/cloud-sdk'
+            },
+            {
+              label: 'SAP Cloud SDK on SAP Answers',
+              href: 'https://answers.sap.com/tags/73555000100800000895'
+            },
+            {
+              label: 'SAP Cloud SDK on SAP Help',
+              href: 'https://help.sap.com/viewer/product/SAP_CLOUD_SDK/1.0/en-US?task=discover_task'
```

Comment:
[req] Not really related to your PR, but can we add this?
```suggestion
            {
              label: 'SAP Cloud SDK on SAP Help',
              href: 'https://help.sap.com/viewer/product/SAP_CLOUD_SDK/1.0/en-US?task=discover_task'
            },
            {
              label: 'Feedback',
              href: 'https://sapinsights.eu.qualtrics.com/jfe/form/SV_0P69X6kJ0Pdxqbc'
```